README.md

```diff
@@ -241,11 +245,21 @@ Specify each parameter using the `--set key=value[,key=value]` argument to `helm
 ```bash
 $ helm install --name my-release \
   --set persistence.efs.enabled=true \
-    alfresco-incubator/alfresco-infrastructure
+    alfresco-incubator/alfresco-infrastructure \
 ```
 
 Alternatively, a YAML file that specifies the values for the parameters can be provided while installing the chart. For example,
 
 ```console
 $ helm install alfresco-incubator/alfresco-infrastructure --name my-release -f values.yaml
 ```
+
+You have also the posibility to deploy only the ingress chart or any other component by itself if needed, using the following:
```

Comment:
Not sure about this section if I should add it here. I will remove it if is not ok

Reply:
Should we add the "component.enabled" parameters to the configuration parameter chart above? Then this section would possibly not be needed.

Reply:
Yes, that will clear up my doubts regarding the options on how to deploy ingress or alfresco-identity-service.
I will do that!